### PR TITLE
[DD-544] Permanent Push: Guided Onboarding - Python & Node

### PR DESCRIPTION
**Pull request by tshen815**
[Ticket](https://circleci.atlassian.net/browse/DD-544)
[Preview](http://circleci-doc-preview.s3-website-us-east-1.amazonaws.com/DD-544/promote-language-guides-preview/?force-all)
**NOTE:** to ensure that you see all the features as intended please make sure to add`?force-all` to the end of the URL if its not there

# Changes

- clean up experiment logic in existing files
- remove files used for experiment
- clean up styles for language guides 

# Reasons
This experiment was a success so we are making it a permanent fixture in our code base 

# Considerations
For Python, in the control group, ~55% made it to the last sections and in the treatment group, ~64% made it to the last sections giving ~9% lift. For Node, in the control group, ~2% made it to the last sections and in the treatment group, ~50% made it to the last sections giving ~42% lift. Also, in section 3, the drop-off rate was reduced from 80% to 13%.

Reference:  https://docs.google.com/spreadsheets/d/1Gr90C9_Sas7v90H1ajy5WtKIE9fgl-axoxfeOx6WWNc/edit#gid=0

### CODE REVIEW

**Review by Mayuko-Sa — APPROVED** [review]

**Review by chantelle11 — APPROVED** [review]
Works for me!

File: jekyll/_includes/doc-footer.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
-{% if page.layout == "classic-docs-experimental" or page.page-type == "homepage" %}
-	{% assign class = "loading-deferred" %}
-{% endif %}
+<!-- {% if page.page-type == "homepage" %}
+{% assign class = "loading-deferred" %}
+{% endif %} -->
```
Comment (rsylvian) [inline jekyll/_includes/doc-footer.html:3]:
```suggestion
```

File: jekyll/_layouts/classic-docs.html [hunk 2]
```diff
@@ -95,7 +95,5 @@ <h5 class="toc-heading">On this page</h5>
   {% include_cached js-assets.html %}
 </div><!-- .outter -->
 {% include popup_cta.html %}
-
-{% include_cached no-script.html %}
```
Comment (rsylvian) [inline jekyll/_layouts/classic-docs.html:99]: are we sure we need to remove this? Is it not going to break our future getting started experiment?

File: src/js/utils.js [hunk 3]
```diff
@@ -11,15 +11,6 @@ export function isUnsupportedBrowser() {
...
 
-const displayElement = (el, display) => {
-  if (el) {
-    el.style.display = display;
-  }
-};
-
-export const displayInitialElement = (el) => displayElement(el, 'initial');
-export const displayBlockElement = (el) => displayElement(el, 'block');
-
```
Comment (rsylvian) [inline src/js/utils.js:22]: same question here

**Pull request merged**